packages/block-library/src/comments-pagination-next/block.json

```diff
@@ -12,7 +12,13 @@
 			"type": "string"
 		}
 	},
-	"usesContext": [ "postId", "comments/perPage", "comments/paginationArrow" ],
+	"usesContext": [
+		"postId",
+		"comments/perPage",
+		"comments/order",
+		"comments/inherit",
+		"comments/defaultPage"
+	],
```

Comment:
"comments/paginationArrow" is missing 😅

Reply:
Nice catch @c4rl0sbr4v0, it's fixed now. ✅